website/pages/docs/nia/installation/configuration.mdx

```diff
@@ -59,7 +59,7 @@ Top level options are reserved for configuring Consul-Terraform-Sync.
 
 ```hcl
 log_level = "INFO"
-port = 8505
+port = 8550
```

Comment:
Can you change this to the default port to show as an example of how to set the port? Having this as a non-default value may be misinterpreted as this the suggested non-default config setup.

Reply:
Sure! Interestingly, I had interpreted this the opposite way: writing the value in a config file suggested it was not a default config value

Reply:
oh yeah. that's true too. I'm not sure tbh but you can keep as is

Reply:
It's ok, I changed it. it's in the docs lower down anyways if it becomes confusing